api/test/gateways/Academy-Benefits/FetchCustomerDb.test.js

```diff
@@ -0,0 +1,105 @@
+const AcademyBenefitsFetchRecord = require('../../../lib/gateways/Academy-Benefits/FetchCustomerDB');
+
+describe('AcademyBenefitsFetchRecord gateway', () => {
+  let db;
+  let logger;
+  const dbError = new Error('Database error');
```

Comment:
Did you need to create the error in this test when you don't test the response to errors in this set?

Reply:
I'm a bit confused with the question, do you mind explaining a bit more what you mean?

Reply:
Have removed it, thanks for spotting it out